src/components/Friends.js

```diff
@@ -6,35 +6,53 @@ class Friends extends React.Component {
     constructor() {
         super();
         this.state = {
-            friends: []
+            friends: [],
+            activeFriend: []
         }
     }
 
     componentDidMount() {
+
         axios.get('http://localhost:5000/friends')
-        .then(res => this.setState({friends: res.data}) )
+        .then(res => {console.log(res); this.setState({friends: res.data}) })
+        .catch(err => err);
       }
+    
```

Comment:
Line 17 - I would recommend properly adding line space and indentation for your `.then` callback (considering that it has multiple statements) for readability.